.travis.yml

```diff
@@ -17,6 +17,7 @@ env:
    - DJANGO="Django<1.9"
    - DJANGO="Django<1.10"
    - DJANGO="Django<1.11"
+   - DJANGO="Django<1.12"
```

Comment:
I couldn't find anywhere in [Django documentation](https://docs.djangoproject.com/en/1.11/) saying release 1.12 is being made. Don't you think it's a little bit too early for this?